discord/ext/commands/context.py

```diff
@@ -322,3 +324,8 @@ async def send_help(self, *args):
                 return None
         except CommandError as e:
             await cmd.on_help_command_error(self, e)
+
+    async def reply(self, *args, **kwargs):
+        return self.message.reply(*args, **kwargs)
```

Comment:
```suggestion
        return await self.message.reply(*args, **kwargs)
```

Reply:
This change does nothing. The original code returned the un-awaited `Message.reply()` coroutine which is arguably better.

Reply:
> This change does nothing. The original code returned the un-awaited `Message.reply()` coroutine which is arguably better.

But it in itself is a coroutine. Which means reply must be awaited, `await ctx.reply()` would then provide you with a coroutine... it needs to either be a coroutine and await the other coroutine, or not be a coroutine and return the coroutine.